assets/css/custom/custom-layout-buttons.scss

```diff
@@ -10,6 +14,11 @@
 
 /* Buttons - used across multiple pages */
 
+$dark-orange-colour: #d35400;
```

Comment:
I think it could be a good idea to extract all the colors to one file, and we can use it anywhere. In Android I used to have a `colors-global`, `colors-screen-main`, `colors-screen-X/Y`. With that you can have all the colors together, and is easier to reuse instead of declare new ones.